Applications/VGG/jni/main.cpp

```diff
@@ -2,12 +2,12 @@
 /**
  * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
  *
- * @file	main.cpp
- * @date	05 Oct 2020
- * @see		https://github.com/nnstreamer/nntrainer
- * @author	Jijoong Moon <jijoong.moon@samsung.com>
- * @bug		No known bugs except for NYI items
- * @brief	This is VGG Example with
+ * @file   main.cpp
+ * @date   05 Oct 2020
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug	   No known bugs except for NYI items
+ * @brief  This is VGG Example with
```

Comment:
<img width="627" alt="Screen Shot 2021-04-27 at 5 13 10 PM" src="https://user-images.githubusercontent.com/33684401/116208673-1da3f600-a77c-11eb-9229-c455bfce9ce5.png">

.... and some more. Looks like the indent is inconsistent in GitHub, but it is okay in IDE.